src/stories/Library/material-grid/MaterialGrid.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import {
+  RecommendedMaterial,
+  RecommendedMaterialProps,
+} from "../recommended-material/RecommendedMaterial";
+
+export type MaterialGridProps = {
+  title: string;
+  amountOfMaterialsToShow: 4 | 8 | 12 | 16 | 20 | 24 | 28 | 32;
+  materials: RecommendedMaterialProps[];
+};
+
+export const MaterialGrid: React.FC<MaterialGridProps> = ({
+  title,
+  amountOfMaterialsToShow,
+  materials,
+}) => {
+  // The JS code here is just an example of how this works.
+  // The proper implementation is in the react repo.
+  const [showAllMaterials, setShowAllMaterials] = useState(false);
+  const [displayedItemCount, setDisplayedItemCount] = useState(8);
+  const showMoreButton = materials.length > 8;
```

Comment:
Consider your use of the value 8.

- Here the number 8 is used specifically. The ticket says that the default is 4. Figma shows 8 (but without a "Show more" button. I wonder what is right.

Reply:
I understand. I've refactored the code and the naming of variables. 

From my understanding the initial max before showing a button should be 8, but i see they mentioned 4 in the ticket now. 

Change it to be using 4 now, and updated the figma link to the the specific component that fits description.